### PR TITLE
Move off of docker/engine-api

### LINKED ISSUE #1743: Move off of docker/engine-api
It is depreciated: https://github.com/docker/engine-api

### PR DESCRIPTION
**Pull request by dashpole**
fix #1743

### CODE REVIEW

File: container/docker/docker.go [hunk 1]
```diff
@@ -49,7 +48,7 @@ func StatusFromDockerInfo(dockerInfo dockertypes.Info) v1.DockerStatus {
 	out.Hostname = dockerInfo.Name
 	out.RootDir = dockerInfo.DockerRootDir
 	out.Driver = dockerInfo.Driver
-	out.ExecDriver = dockerInfo.ExecutionDriver
+	out.ExecDriver = ""
```
Comment (tallclair) [inline container/docker/docker.go]: nit: just delete this

Comment (dashpole) [reply container/docker/docker.go]: done.

**Comment by dashpole** [comment]
I would rather just stick with docker/docker.  Who knows what will happen to godeps if we change it

**Pull request merged**